Source/WebCore/page/Page.h

```diff
@@ -1309,6 +1310,9 @@ class Page : public RefCountedAndCanMakeWeakPtr<Page>, public Supplementable<Pag
     WEBCORE_EXPORT void setPresentingApplicationBundleIdentifier(String&&);
 #endif
 
+    WEBCORE_EXPORT void allowHDR();
```

Comment:
Super confusing to have two methods called "allowHDR". Also the interaction between "allow" and "suppress" is unclear.